apps/pay/app/lnurlp/[username]/callback/route.ts

```diff
@@ -114,9 +116,13 @@ export async function GET(
     })
   }
 
+  // eslint-disable-next-line
+  // @ts-ignore
```

Comment:
why is this required? please use declare or try to find the types

Reply:
No longer required